tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -14,6 +14,10 @@ namespace Datadog.Trace.ClrProfiler.Managed.Loader
     public partial class Startup
     {
         private const string AssemblyName = "Datadog.Trace, Version=2.0.1.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb";
+        private const string AzureAppServicesKey = "DD_AZURE_APP_SERVICES";
+        private const string AasCustomTracingKey = "DD_AAS_ENABLE_CUSTOM_TRACING";
+        private const string AasCustomMetricsKey = "DD_AAS_ENABLE_CUSTOM_METRICS";
+        private const string TraceEnabledKey = "DD_TRACE_ENABLED";
```

Comment:
`DD_AZURE_APP_SERVICES` is already defined in `AzureAppServices`, and `DD_TRACE_ENABLED` is already defined in `ConfigurationKeys`. I suggest we reuse those, and add `DD_AAS_ENABLE_CUSTOM_TRACING` and `DD_AAS_ENABLE_CUSTOM_METRICS` to `AzureAppServices`.

Reply:
Ah, good point 👍